tests/v5/mqtt5_client_tests.c

```diff
@@ -97,16 +97,9 @@ void s_publish_received_callback(const struct aws_mqtt5_packet_publish_view *pub
     (void)user_data;
 }
 
-AWS_STATIC_STRING_FROM_LITERAL(s_client_id, "HelloWorld");
+AWS_STRING_FROM_LITERAL(g_default_client_id, "HelloWorld");
```

Comment:
This is set here but not in mqtt3_to_mqtt5_adapter_tests.c. g_default_client_id is extern const and I'm not sure if that means setting this here will insure it's set when the adapter tests run.

Reply:
extern is a declaration not a definition